cmd/tuf/main.go

```diff
@@ -127,13 +126,6 @@ func getPassphrase(role string, confirm bool) ([]byte, error) {
 		return []byte(pass), nil
 	}
 
-	state, err := term.SaveState(0)
-	if err != nil {
-		return nil, err
-	}
-	term.DisableEcho(0, state)
-	defer term.RestoreTerminal(0, state)
-
```

Comment:
Could switch to the `golang.org/x/crypto/ssh/terminal` package that has a [`ReadPassword`](https://godoc.org/golang.org/x/crypto/ssh/terminal#ReadPassword) function?

Reply:
Or the seemingly more convenient [`Terminal.ReadPassword`](https://godoc.org/golang.org/x/crypto/ssh/terminal#Terminal.ReadPassword) that includes the prompt too.

Reply:
Oh, nice. Will do.